Resources/Prototypes/_NF/Language/languages.yml

```diff
@@ -0,0 +1,525 @@
+  - Dragon
+  - Ninja
+  - Secret
+
+# A mess of broken Japanese, spoken by Felinds and Oni
+- type: language
+  id: Nekomimetic
+  obfuscateSyllables: true
+  replacement:
+  - neko
+  - nyan
+  - mimi
+  - moe
+  - mofu
+  - fuwa
+  - kyaa
+  - kawaii
+  - poka
+  - munya
+  - puni
+  - munyu
+  - ufufu
+  - icha
+  - doki
+  - kyun
+  - kusu
+  - nya
+  - nyaa
+  - desu
+  - kis
+  - ama
+  - chuu
+  - baka
+  - hewo
+  - boop
+  - gato
+  - kit
+  - sune
+  - yori
+  - sou
+  - baka
+  - chan
+  - san
+  - kun
+  - mahou
+  - yatta
+  - suki
+  - usagi
+  - domo
+  - ori
+  - uwa
+  - zaazaa
+  - shiku
+  - puru
+  - ira
+  - heto
+  - etto
+
+# Spoken by the Lizard race.
+- type: language
+  id: Draconic
+  obfuscateSyllables: true
+  replacement:
+  - za
+  - az
+  - ze
+  - ez
+  - zi
+  - iz
+  - zo
+  - oz
+  - zu
+  - uz
+  - zs
+  - sz
+  - ha
+  - ah
+  - he
+  - eh
+  - hi
+  - ih
+  - ho
+  - oh
+  - hu
+  - uh
+  - hs
+  - sh
+  - la
+  - al
+  - le
+  - el
+  - li
+  - il
+  - lo
+  - ol
+  - lu
+  - ul
+  - ls
+  - sl
+  - ka
+  - ak
+  - ke
+  - ek
+  - ki
+  - ik
+  - ko
+  - ok
+  - ku
+  - uk
+  - ks
+  - sk
+  - sa
+  - as
+  - se
+  - es
+  - si
+  - is
+  - so
+  - os
+  - su
+  - us
+  - ss
+  - ss
+  - ra
+  - ar
+  - re
+  - er
+  - ri
+  - ir
+  - ro
+  - or
+  - ru
+  - ur
+  - rs
+  - sr
+  - a
+  - a
+  - e
+  - e
+  - i
+  - i
+  - o
+  - o
+  - u
+  - u
+  - s
+  - s
+
+# Spoken by the Vulpkanin race.
+- type: language
+  id: Canilunzt
+  obfuscateSyllables: true
+  replacement:
+  - rur
+  - ya
+  - cen
+  - rawr
+  - bar
+  - kuk
+  - tek
+  - qat
+  - uk
+  - wu
+  - vuh
+  - tah
+  - tch
+  - schz
+  - auch
+  - ist
+  - ein
+  - entch
+  - zwichs
+  - tut
+  - mir
+  - wo
+  - bis
+  - es
+  - vor
+  - nic
+  - gro
+  - lll
+  - enem
+  - zandt
+  - tzch
+  - noch
+  - hel
+  - ischt
+  - far
+  - wa
+  - baram
+  - iereng
+  - tech
+  - lach
+  - sam
+  - mak
+  - lich
+  - gen
+  - or
+  - ag
+  - eck
+  - gec
+  - stag
+  - onn
+  - bin
+  - ket
+  - jarl
+  - vulf
+  - einech
+  - cresthz
+  - azunein
+  - ghzth
+
+# The common language of the Sol system.
+- type: language
+  id: SolCommon
+  obfuscateSyllables: true
+  replacement:
+  - tao
+  - shi
+  - tzu
+  - yi
+  - com
+  - be
+  - is
+  - i
+  - op
+  - vi
+  - ed
+  - lec
+  - mo
+  - cle
+  - te
+  - dis
+  - e
+
+# Languages spoken by various critters.
+- type: language
+  id: Cat
+  obfuscateSyllables: true
+  replacement:
+  - murr
+  - meow
+  - purr
+  - mrow
+
+- type: language
+  id: Dog
+  obfuscateSyllables: true
+  replacement:
+  - woof
+  - bark
+  - ruff
+  - bork
+  - raff
+  - garr
+
+- type: language
+  id: Mothroach
+  obfuscateSyllables: false
+  replacement:
+  - Chitter
+  - Buzz
+  - Chirp
+  - Squeak
+  - Peep
+  - Eeee
+  - Eep
+
+- type: language
+  id: Xeno
+  obfuscateSyllables: true
+  replacement:
+  - sss
+  - sSs
+  - SSS
+
+- type: language
+  id: RobotTalk
+  obfuscateSyllables: true
+  replacement:
+  - beep
```

Comment:
Additional robot talk replacements could be:
1, 11, 111, 0, 00, 101, 1001, etc. in SS13 borgs and AI speak binary whereas machine people speak trinary.

Reply:
Yeah, we could do that.